.vscode/sample.launch.json

```
@@ -51,13 +57,10 @@
         "${workspaceFolder}/api/tests"
       ],
       "internalConsoleOptions": "openOnSessionStart",
-      "skipFiles": [
-        "<node_internals>/**",
-        "api/tests/test-helper.js"
-      ]
+      "skipFiles": ["<node_internals>/**", "api/tests/test-helper.js"]
     },
     {
-      "name": "api mocha current file",
+      "name": "api mocha current file (equivaut à @doc)",
```

Pour rester dans la même langue :)

```suggestion
      "name": "api mocha current file (similar to @doc)",
```

---

Committed